features/collection/components/MetaTags.tsx

```diff
@@ -13,7 +14,24 @@ export function MetaTags({ collection }: MetaTagsProps) {
       searchParams.query.length > 0 || searchParams.filters.allIds.length > 0;
    return (
       <Head>
-         {isFiltered && <meta name="robots" content="noindex,nofollow" />}
+         {isFiltered ? (
+            <meta name="robots" content="noindex,nofollow" />
+         ) : (
+            <>
+               <link
+                  rel="canonical"
+                  href={`https://ifixit.com/collections/${collection.handle}${
```

Comment:
Make sure to use www since it will redirect to www if you don't.
```suggestion
                  href={`https://www.ifixit.com/collections/${collection.handle}${
```

Reply:
Applied your suggestion. I've just realized that probably we want this url to come from Strapi `Store` settings, so that when we use the Nextjs app for the other store this will work out of the box. I'll do this on another PR